internal/drivers/nomad/driver.go

```diff
@@ -768,3 +794,15 @@ func generateDestroyCommand(vm string) string {
 		fi
 	`, ignitePath, vm, ignitePath, vm, ignitePath, vm, ignitePath, vm)
 }
+
+// Request Nomad to assign available ports dynamically.
+// 1 for lite engine and another n ports for gitspaces as requested
+// Since the port labels have to be unique, VM ID is used usually.
+// For gitspaces port, prefix of _gitspaces_{port_number_count} is added to the label to keep it unique
+func getNetworkResources(portLabel string, portsCount int) []*api.NetworkResource {
```

Comment:
Nit: could we rename the fields to make it more explicit what is being passed here?
`portLabel` -> `liteEnginePortLabel`
`portsCount` -> `gitspacesPortCount`
Also - just curious - what do we use the multiple git spaces ports for?

Reply:
It's not just liteEnginePortLabel right? It's used for the other 2 ports as well
We need one port for the gitspaces agent
And second port for either SSH or web (depending on the IDE type chosen vscode/vscode web)